00-intro.md

```diff
@@ -64,13 +64,13 @@ Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
 Using Bash or any other shell
-sometimes feels more like programming than like using a mouse.
+is more like programming than like using a mouse.
 Commands are terse (often only a couple of characters long),
 their names are frequently cryptic,
 and their output is lines of text rather than something visual like a graph.
 On the other hand,
 the shell allows us to combine existing tools in powerful ways with only a few keystrokes
-and to set up pipelines to handle large volumes of data automatically thus improving productivity and reproducibility.
```

Comment:
but my impression is that this sentence refers to writing scripts which handle large data. Using pipeline is covered in the first part of this sentence:

> shell allows us to **combine** existing tools in powerful ways